### ISSUE TITLE
Accessibility Headphone Accommodations on MacOS

### ISSUE THREAD

**Issue by beachballing** [beachballing/rotten#3]
`MacBook Pro 14-inch 2021` `MacOS 14.2.1` `AirPods Pro 2`

The have an "Headphone Accommodations" section in the accessibility features on iOS.
I'm a user of these accommodations to enhance the sound of my AirPods Pro 2.
On MacOS however this feature cannot be found anywhere in the accessibility features or system settings.
